app/controllers/courses_controller.rb

```diff
@@ -81,21 +81,22 @@ def create
       if @course.save
         format.html { redirect_to(@course, notice: 'Course was successfully created.') }
       else
-        format.html { render action: "new" , notice: 'Course could not be created.' }
+        format.html { render action: 'new', notice: 'Course could not be created.' }
       end
     end
   end
 
-private
+  private
+
   def course_params
     params.permit(course: [:name, :source_url, :git_branch])
   end
 
   def assign_show_view_vars
-    @exercises = @course.
-      exercises.
-      includes(:course).
-      select {|ex| ex.visible_to?(current_user) }.natsort_by(&:name)
+    @exercises = @course.exercises
+                   .includes(:course)
+                   .select { |ex| ex.visible_to?(current_user) }
+                   .natsort_by(&:name)
```

Comment:
Could these cases be made to indent starting from the beginning of the first line instead of the beginning of the expression? So this would become:
```
    @exercises = @course.exercises
      .includes(:course)
      .select { |ex| ex.visible_to?(current_user) }
      .natsort_by(&:name)
```

Reasons: it's and easy to forget chore to realign when changing the first statement, and it needlessly reduces the width available for the indented lines.